iphone/cli/hooks/frameworks.js

```diff
@@ -57,7 +64,12 @@ class FrameworkManager {
 			post: (builder, callback) => {
 				this._logger.trace('Starting third-party framework detection');
 				this._builder = builder;
-				this.detectFrameworks().then(callback, callback);
+				this.detectFrameworks().then(callback, e => {
```

Comment:
<!--
  0 failure: 
  2 warning:  iphone/cli/hooks/..., iphone/cli/hooks/...
  
  
  DangerID: danger-id-Danger;
  File: iphone/cli/hooks/frameworks.js;
  Line: 67;
-->

- :warning: iphone/cli/hooks/frameworks.js line 67 – Avoid calling back inside of a promise. (promise/no-callback-in-promise)
- :warning: iphone/cli/hooks/frameworks.js line 67 – Expected catch() or return (promise/catch-or-return)